apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/DoedshendelseService.kt

```diff
@@ -34,6 +36,24 @@ class DoedshendelseService(
 
     fun kanBrukeDeodshendelserJob() = featureToggleService.isEnabled(DoedshendelseFeatureToggle.KanLagreDoedshendelse, false)
 
+    private fun DoedshendelseInternal.erUlikPdlDoedshendelse(doedshendelse: PdlDoedshendelse): Boolean {
+        return this.endringstype != doedshendelse.endringstype ||
+            this.avdoedDoedsdato != doedshendelse.doedsdato
```

Comment:
Hva hvis det er en annen endringstype, og dødsdato er ulik, f. eks? Da vil det nå opprettes nye innslag av dødshendelser på de berørte, og ikke oppdatere de eksisterende hendelsene. Vi burde vel egentlig oppdatere i de tilfellene?

Reply:
Hadde vært digg om vi vente med å mekke berørte fnr til vi faktisk skal sende det ut

Reply:
Joda, vi skal sende brev (og opprette oppgave) ved samtidig dødsfall. Det jeg tenkte på her, er at dersom det for eksempel kommer inn en korrigering på dødsdato, så vil vi opprette et nytt innslag, i stedet for å oppdatere den vi allerede har?

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3924/commits/0c4d1c3539df0c36cf491e1f0378c3c9d13e6401